homeassistant/components/sensor/envirophat.py

```diff
@@ -53,7 +53,13 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the Sense HAT sensor platform."""
-    data = EnvirophatData(config.get(CONF_USE_LEDS))
+    try:
```

Comment:
@pvizeli importing the library is used to check whether the physical device is actually available, as per @balloob's comment in https://github.com/home-assistant/home-assistant/pull/7438#discussion_r114822526

If I've misunderstood something, would be greatful for some comments on how the library should be imported and the test performed if not like this.